src/ansys/meshing/prime/autogen/connectstructs.py

```diff
@@ -908,7 +908,7 @@ def set_default(
         ignore_face_zonelets: Iterable[int], optional
             Face zonelet ids that subtract volumes should not remove (for example, periodic or fluid cap zonelets). If ignore face zonelets are provided, then the target volumes after subtract operation need to be recomputed.
```

Comment:
```suggestion
            Face zonelet IDs that subtract volumes should not remove (for example, periodic or fluid cap zonelets). If values are provided for the ``ignore_face_zonelets`` parameter, then the target volumes after the subtract operation must be recomputed.
```
In descriptive text, "ids" shoud always be "IDs".